src/prompt/parent_branches.go

```diff
@@ -37,20 +37,25 @@ The latter allows to build on top of currently unshipped features.
 
 `
 var parentBranchPromptTemplate = "Please specify the parent branch of '%s':"
+var perennialBranchOption = "<none> (perennial branch)"
 
 func askForBranchAncestry(branchName string) {
 	current := branchName
-	choices := git.GetLocalBranches()
+	choices := git.GetLocalBranchesWithMainBranchFirst()
```

Comment:
Put this back to having main first so the options will be 
```
<none> (perennial branch)
main 
other branches in alphabetical order
```
with main the preselected option. Wanted to make users aware that none is an option and if the main branch was with all the other branches in alpha order, you may not see the none option

Reply:
I like this. That way the user can go down to select branches, or up to select no parent branch.